app/controllers/cuisines_controller.rb

```diff
@@ -0,0 +1,15 @@
+# app/controllers/cuisines_controller.rb
+class CuisinesController < ApplicationController
+  def index
+    @q = Cuisine.ransack(params[:q])
+    @cuisines = if params[:q].present?
+                  @q.result(distinct: true)
+                else
+                  Cuisine.all
+                end
```

Comment:
```suggestion
    @cuisines = if params[:q].present?
                  @q.result(distinct: true)
                else
                  Cuisine
                end.newest
```
`newest` là scope sắp xếp nhé
ngoài ra, aloo, index mà ko thấy phân trang hì